### ISSUE TITLE
Quick edit percentage complete

### ISSUE THREAD

**Issue by dugite-code** [dmfs/opentasks#230]
It would be great to quickly change the percentage complete when viewing a task rather than having to opening the edit view and scrolling to the bottom much like how you can add new items to the checklist

**Comment by dmfs** [comment]
indeed. But we'll probably disable that if there is a non-empty checklist, since the progress is determined by the checklist if there is any.
